docs/components/operate/userguide/resolve-incidents-update-variables.md

```diff
@@ -7,7 +7,7 @@ description: "Let's examine variable and incidents."
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-Every process instance created for the process model used in the [getting started tutorial](./guides/getting-started/model-your-first-process.md) requires an `orderValue` so the XOR gateway evaluation will happen properly.
+Every process instance created for the [`order-process.bpmn`](https://docs.camunda.io/assets/files/order-process-2ae29e9d889a3d640464be250206d550.bpmn/) process model requires an `orderValue` so the XOR gateway evaluation will happen properly.
```

Comment:
Potentially blocking but probably a gap in my understanding: 

Is this the correct URL for this image? I'm surprised to see it with an absolute path instead of relative...

But then I also don't even see this file in (a) this PR or (b) master. I suspect there might be some infrastructure that I don't understand that makes it work -- does this path resolve to something not in our source?

Reply:
I think you are right @pepopowitz, we shouldn't use absolute links include in the domain for resources. The resource should be relative to the domain, and I think docusaurus is also generating this hash at the end of the assets, not sure if it changes between builds. I remember fighting with this a while ago, but can remember right now where it was.

Reply:
But I just checked the source and apparently this is something we already do "wrong" since docs 1.3 version. So nothing new introduced in this PR, we can maybe create a follow-up issue to address this in a separate PR to not block this one.

Reply:
What's interesting is this is also how it is referenced in `basic-operate-navigation`. A good question for @akeller when she is back online, because I don't see this in master either. I didn't make this change so I can't directly say why/why not it isn't structured as an absolute link.

Reply:
I'm still digging out of my notifications after vacation - did this get turned into an issue for us to audit and fix absolute links across our docs? In this instance, I wonder if tabbed behavior (either currently or previously) required this.

Reply:
@akeller I'll launch an issue shortly so we can continue conversation in there 👍

Reply:
https://github.com/camunda/camunda-platform-docs/issues/968